tests/test_gaussian.py

```diff
@@ -65,7 +65,7 @@ def test_unsupported_gates(self):
         for g in all_gates - gates:
             op = getattr(qml.ops, g)
 
-            if op.num_wires == 0:
+            if op.num_wires <= 0:
```

Comment:
just curious, why are these changed?

Reply:
Originally PennyLane operations either specified a specific number of required wires, or 'all wires' (`Operation.num_wires = 0`).

In practice though, this didn't work, as there existed operations that didn't fit into these two categories --- namely operations that work on 'any' wires (`QubitStateVector`, `BasisState`, `GaussianState`).

In https://github.com/XanaduAI/pennylane/pull/277, we created enumerations for `Any` and `All` wires to fix this. For backwards compatibility, `All == 0`, and `Any = -1`.

This test is marked for deprecation, so this is just a quick fix to ensure it continue to pass.

Reply:
That's what I figured. :+1: